### PR TITLE
[lambda][output] request_helper changes and phantom output

### PR DESCRIPTION
**Pull request by armtash**
to @airbnb/streamalert-maintainers

size: small

* Added an output handler for Phantom.
* REST data collector needs to be configured in Phantom.
* Secret should be structured and encrypted exactly like slack,
* https://PHANTOMURL,YOUR-TOKEN-HERE
* Added request_helper optional args "headers" and "verify" to allow 
* output to ignore server certificate checking. Default is verify=True
* Removed request_helper required arg "endpoint" as it's no longer needed.
* request_helper returns its urllib2 response to allow for outputs to
* chain http requests and better validate successful requests.

### CODE REVIEW

**Comment by jacknagz** [comment]
@armtash can you drop the `request_helper` changes commit? It's a dupe from #65

File: stream_alert_output/main.py [hunk 1]
```diff
@@ -244,6 +244,54 @@ def _pagerduty(self, rule_name, alerts):
         if resp and resp.getcode() in range(200,299):
             logger.info('Successfully sent alert to Pagerduty.')
 
+    def _phantom(self, rule_name, alerts):
```
Comment (jacknagz) [inline stream_alert_output/main.py]: This is great! Would you mind also `pylint`ing this new `output`?

**Comment by jacknagz** [comment]
one final request @armtash - please update the PR body/title to reflect both the Phantom addition and the request helper changes

**Comment by jacknagz** [comment]
@armtash any chance on updating/merging this?

**Comment by armtash** [comment]
@jacknagz I think you can merge now.

**Pull request merged**